packages/terra-button/tests/jest/Button.test.jsx

```diff
@@ -56,9 +57,9 @@ it('should render a Button with merged attributes', () => {
   /* eslint-enable react/forbid-component-props */
 });
 
-it('should render as an anchor when provided an href', () => {
-  const button = shallow(<Button text="href" href="MockHref" />);
-  expect(button).toMatchSnapshot();
+it('should render as an anchor when AnchorButton is used', () => {
```

Comment:
Instead of updating Button.test.jsx. New test file similar to icon-button would be easier to add new tests for anchor button

Reply:
Added new test file in https://github.com/cerner/terra-core/pull/3655/commits/81401caf927c1b11a30aa72ed0740f8e7e2c6df6